.gitattributes

```
@@ -3,5 +3,10 @@
 
 /.gitattributes export-ignore
 /.gitignore export-ignore
+.travis.yml
```

does this need a leading `/`?

---

```suggestion
/.travis.yml
```

I honestly don't know if they need them but I keep it there to be safe, will add.

---

yeah unsure as well 🤷‍♂